libmodernize/main.py

```diff
@@ -144,4 +147,9 @@ def main(args=None):
         rt.summarize()
 
     # Return error status (0 if rt.errors is zero)
-    return int(bool(rt.errors))
+    result = bool(rt.errors)
+    # If we are enforcing python 3 compatibility, return a non-zero exit code if we had to modify
+    # any files.
+    if options.enforce:
+        result = result or bool(rt.files)
```

Comment:
Do you want to have a different exit code for this case - e.g. 1 means error, 2 means changes were needed with the `--enforce` flag in use. It seems worthwhile making it possible to detect that difference.